fearless/ApplicationLayer/Services/Balance/WalletBalanceSubscription/WalletBalanceSubscriptionAdapter.swift

```diff
@@ -312,34 +305,18 @@ extension WalletBalanceSubscriptionAdapter: EventVisitorProtocol {
         pricesProvider?.refresh()
     }
 
+    func processSelectedAccountChanged(event _: SelectedAccountChanged) {
+        fetchAllMetaAccounts()
```

Comment:
why we reset all data when select new wallet?

Reply:
Because this event works when we add new wallet?

Reply:
Perhaps it will be better check selected wallet did changed or new wallet was added.

Reply:
If did changed selected wallet needs check what parameters did changed in wallet, and think should be reset or not